### PR TITLE
plug memory leak

### PR DESCRIPTION
**Pull request by boxerab**
Found this with cppcheck.

### CODE REVIEW

File: src/runtime_src/core/pcie/emulation/hw_em/zynqu/shim_hwem.cpp [hunk 1]
```diff
@@ -151,6 +151,7 @@ int ZYNQShim::xclLoadXclBin(const xclBin *header) {
   cmd->addArg(xclBinName.c_str());
   uint32_t length;
   uint8_t* buff = cmd->generateBuffer(&length);
+  delete cmd;
```
Comment (stsoe) [inline src/runtime_src/core/pcie/emulation/hw_em/zynqu/shim_hwem.cpp:154]: Yes, indeed.  Let fix is properly and use std::unique_ptr.

Comment (boxerab) [reply src/runtime_src/core/pcie/emulation/hw_em/zynqu/shim_hwem.cpp]: Good idea. I've made the change

**Comment by stsoe** [comment]
ok to test

**Comment by boxerab** [comment]
Thanks. cppcheck highly recommended :)

**Comment by abalasa** [comment]
retest this please

**Pull request merged**